yaksh/models.py

```diff
@@ -402,6 +402,9 @@ def read_json(self, file_path, user, files=None):
             with open(json_file, 'r') as q_file:
                 questions_list = q_file.read()
                 self.load_questions(questions_list, user, file_path, files)
+            return "Questions Uploaded Successfully"
+        else:
+            return "Please upload zip file with questions_dump.json in it."
```

Comment:
Hey incase of uploading a file with a bad file name, i.e. anything except filename questions_dump.json, will raise an exception. Hence this error message return is pretty useless.